src/page_body/structure/blocks/page_block_component_checklist.pr

```diff
@@ -13,7 +13,7 @@
 {[ let hasNonBooleanPropertiesToShow = false /]}
 
 {* Render content health block *}
-{[ if (componentId && (hasBooleanPropertiesToShow || hasNonBooleanPropertiesToShow)) ]}
+{[ if componentId ]}
```

Comment:
`hasBooleanPropertiesToShow` and `hasNonBooleanPropertiesToShow` are calculated further so they are always `false` at this line